ProgressBar: update demo description + minor code fix (#2124)
DevExpress/devextreme-demos: JSDemos/Demos/ProgressBar/Overview/description.md

This demo shows how to configure the ProgressBar component.

> To create a ProgressBar, declare it in markup. You can specify the following properties to change the ProgressBar numeric scale:

Reviewer: ```suggestion
To create a ProgressBar, declare it in markup. You can specify the following properties to change the ProgressBar's numeric scale:
```

- [min](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#max)    
The min and max properties limit the range of accepted values. 

- [value](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#value)    
This property specifies the current value. If you want to switch the ProgressBar to an indeterminate state, set the [value](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#value) property to `false`.

> This demo uses the custom timer function to increase the ProgressBar value.

Reviewer: ```suggestion
This demo uses the custom timer function to increase the ProgressBar's value.
```

> When the ProgressBar reaches the maximum value, the [complete](/Documentation/ApiReference/UI_Components/dxProgressBar/Events/#complete) event is raised. Use the [onComplete](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#onComplete) function to handle it.

Reviewer 2: Avoidable passive voice.

```suggestion
When the ProgressBar reaches the maximum value, the [complete](/Documentation/ApiReference/UI_Components/dxProgressBar/Events/#complete) event occurs. Use the [onComplete](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#onComplete) function to handle it.
```

> The progress status displays a ratio between the current and maximum values and indicates the progress made. Use the [showStatus](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#showStatus) property to display or hide the status string. To format the status string, use the [statusFormat](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#statusFormat) function.

Reviewer: ```suggestion
The progress status displays a ratio between the current and maximum values and indicates the progress. Use the [showStatus](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#showStatus) property to display or hide the status string. To format the status string, use the [statusFormat](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#statusFormat) function.
```

> The progress status displays a ratio between the current value and current value that indicates the made progress. Use the [showStatus](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#showStatus) property to change the status visibility. To format the status string, use the [statusFormat](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#statusFormat) function.

Reviewer 2: > The progress status displays a ratio between the current value and current value that indicates the made progress.

Something is amiss here. Inspect this sentence closely, for it makes no sense in its current state.

> Use the [showStatus](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#showStatus) property to change the status visibility.

to change the status visibility == to display or hide the status string?

Author: I am sorry, I will rewrite the paragrpah

Reviewer 2: Thank you.

> The progress status displays a ratio between the current value and maximum value and indicates the made progress. Use the [showStatus](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#showStatus) property to display or hide the status string. To format the status string, use the [statusFormat](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#statusFormat) function.

Reviewer 2: > a ratio between the current value and **maximum value**

An article is missing. Besides, we can join "current" and "maximum" under "value".

the made progress -> the progress made.

```suggestion
The progress status displays a ratio between the current and maximum values and indicates the progress made. Use the [showStatus](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#showStatus) property to display or hide the status string. To format the status string, use the [statusFormat](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#statusFormat) function.
```
